auth0_flutter/EXAMPLES.md

```
@@ -1251,6 +1253,84 @@ final credentials = await auth0.api.loginWithSmsCode(
+```dart
+final credentials = await auth0.passwordless.loginWithOtp(
+    authSession: challenge.authSession,
+    otp: '123456',
+    audience: 'YOUR_AUTH0_API_IDENTIFIER',
+    scopes: {'openid', 'profile', 'email', 'offline_access'});
+```
+
+</details>
+
+> [!NOTE]
+> If the user has MFA configured, `loginWithOtp` fails with an [`ApiException`](#api-client-errors) whose `isMultifactorRequired` is `true`. Continue the flow using the existing MFA APIs (`auth0.api.multifactorChallenge` / `auth0.api.loginWithOtp`).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)